seed/php-sdk/imdb/src/Core/RawClient.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Seed\Core;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Utils;
+use InvalidArgumentException;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Message\StreamInterface;
+use Seed\ClientOptions;
+
+
+class RawClient
+{
+    /**
+     * The client options used to make requests.
+     *
+     * @var ClientOptions
+     */
+    private ClientOptions $clientOptions;
+
+    /**
+     * The client options used to make requests.
+     *
+     * * @param ClientOptions|null $clientOptions
+     */
+    public function __construct(
+        ?ClientOptions $clientOptions,
+    )
+    {
+        $this->clientOptions = $clientOptions ?? new ClientOptions();
+    }
```

Comment:
nit: I believe that this is equivalent
```suggestion
    /**
     * @param ClientOptions $clientOptions The client options used to make requests.
     */
    public function __construct(
        private readonly ClientOptions $clientOptions = new ClientOptions(),
    )
    {
    }
```

not important right now, but will comment on these things so we can build our php chops together!

Reply:
Nice, just verified this works well - let's go with this!